source/molang/boolean.json

```diff
@@ -3,6 +3,6 @@
   "title": "Molang Boolean",
   "description": "The minecraft molang definition that results in a boolean.",
   "anyOf": [{ "type": "string", "minLength": 0 }, { "type": "boolean" }],
-  "examples": [0, "query.variant", "(1.0)", "query.", "variable.=;"],
+  "examples": ["(1.0)", "(0.0)"],

```

Comment:
Why no more molang examples? its a molang boolean so it should be acceptable 🤔

Reply:
Personally just makes it annoying while trying to make use of the molang completion. query.variant is already there, variable.=; is suggested in places where it's not valid.